### PR TITLE
Defined a new IP protection process for ECIPs.

### PR DESCRIPTION
**Pull request by bobsummerwill**
Defined a new IP protection process for ECIPs.
Mandatory use of Apache 2.0 for ECIPs, plus optional use of various copyleft license for the source code.
Mandatory DCOs for contributions with real name requirement.
Option for ECIP editors to override the real name requirement if they feel a contribution is exceptional.

### CODE REVIEW

**Comment by TokenHash** [comment]
I would leave the license option as they are, implement DCO, but with no identification requirement of developers.

File: _specs/ecip-1000.md [hunk 1]
```diff
@@ -293,64 +293,137 @@ To avoid doubt: comments and status are unrelated metrics to judge an ECIP, and
...
+example source code.  Earlier versions of ECIP-1000 allowed a wider
+range of licenses, but that left loopholes for IP attacks on ETC.
 
-For example, a preamble might include the following License header:
+The preamble for all new ECIPs should look like this one of these two options:
 
-    License: BSD-2-Clause
-             GNU-All-Permissive
+    License: Apache-2.0
+    License: Apache-2.0+
```
Comment (BelfordZ) [inline _specs/ecip-1000.md]:
```suggestion
```

File: _specs/ecip-1000.md [hunk 2]
```diff
@@ -293,64 +293,137 @@ To avoid doubt: comments and status are unrelated metrics to judge an ECIP, and
 
 ### Specification
 
-New ECIPs may be accepted with the following licenses. Each new ECIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respective abbreviation given below.
+All new ECIPs should use Apache 2.0 or newer licensing for the specification and
+may use Apache 2.0+ or various other copyleft licenses for any associated
```
Comment (BelfordZ) [inline _specs/ecip-1000.md]: No. Apache 2.0 only.

File: _specs/ecip-1000.md [hunk 3]
```diff
@@ -293,64 +293,137 @@ To avoid doubt: comments and status are unrelated metrics to judge an ECIP, and
 
 ### Specification
 
-New ECIPs may be accepted with the following licenses. Each new ECIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respective abbreviation given below.
+All new ECIPs should use Apache 2.0 or newer licensing for the specification and
```
Comment (BelfordZ) [inline _specs/ecip-1000.md]: All code contributed to a repository assumes the repositories license. Best to let legal precedent do the talking, and not even mention license at all.

**Comment by BelfordZ** [comment]
@bobsummerwill Thanks for taking the time to look at this file. IMHO the best thing to do here would be to delete the file all together, and add a LICENSE file to the root of the repo, which contains the apache 2.0 license. 

The rest takes care of itself.

**Comment by bobsummerwill** [comment]
I will retire this in favor of collaborating on @BelfordZ's approach on #163.

**Pull request closed without merging**